tools/BUILD

```diff
@@ -13,12 +13,7 @@
 # limitations under the License.
 package(default_visibility = ["//visibility:private"])
 
-py_binary(
+alias(
```

Comment:
This is just to have backwards compatibility with `update_deps.sh`, since the target is private, we could just move `update_deps.sh` to reference the new target at `//container/go/cmd/update_deps`

Reply:
Couldn't we just change `update_deps.sh` to `bazel run //container/go/cmd/update_deps` instead?

Reply:
That does seem much cleaner